react/src/components/DoubleTag.tsx

```diff
@@ -1,19 +1,20 @@
 import Flex from './Flex';
+import TextHighlighter from './TextHighlighter';
 import { Tag } from 'antd';
 import _ from 'lodash';
 import React from 'react';
 
 export type DoubleTagObjectValue = {
-  label: ValueType;
+  label: string;
   color?: string;
+  highlightKeyword?: string;
```

Comment:
It would be better to set highlightKeyword as a prop of DoubleTag instead of a property of values.